Dynatrace/dynatrace-operator
# Adds step to dynamically create opencontainers labels

# Description

Modifies the GitHub CI tasks creating the operator image to include the docker/metadata-action.
This action adds standardized opencontainer labels.

## How can this be tested?

* Use `scopeo` to inspect the snapshot image built with this PR
* Check that all necessary labels exist in the manifest
* Check that the labels values are correct
## Checklist
